getpaid/backends/pagseguro/__init__.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+from decimal import Decimal
+import logging
+import datetime
+from django.core.urlresolvers import reverse
+from django.db.models import get_model
+from django.utils.timezone import utc
+import requests
+import time
+from django.utils.translation import ugettext_lazy as _
+from getpaid.signals import user_data_query
+from getpaid.backends import PaymentProcessorBase
+import urllib
+from xml.dom.minidom import parseString
+
+
+logger = logging.getLogger('getpaid.backends.pagseguro')
+
+# see: https://pagseguro.uol.com.br/v2/guia-de-integracao/api-de-notificacoes.html#v2-item-api-de-notificacoes-status-da-transacao
+class PagseguroTransactionStatus:
+    IN_PROGRESS = 1     # payment transaction started and waiting be paid
+    IN_ANALISE = 2
+    PAID    = 3
+    AVAILABLE = 4
+    IN_DISPUTE = 5
+    REFUNDED = 6
+    CANCELED = 7
+
+
+class PaymentProcessor(PaymentProcessorBase):
+    BACKEND = 'getpaid.backends.pagseguro'
+    BACKEND_NAME = _('PagSeguro')
+    BACKEND_ACCEPTED_CURRENCY = ('BRL', )
+    BACKEND_LOGO_URL = 'getpaid/backends/pagseguro/pagseguro_logo.png'
```

Comment:
I don't know exactly but I suspect we can not use logos and other materials that may be under copy right.